toolchain/sem_ir/formatter.h

```diff
@@ -16,8 +16,17 @@ namespace Carbon::SemIR {
 // Formatter for printing textual Semantics IR.
 class Formatter {
  public:
-  explicit Formatter(const Lex::TokenizedBuffer& tokenized_buffer,
-                     const Parse::Tree& parse_tree, const File& sem_ir);
+  // A callback that indicates whether a specific entity, identified by its
+  // declaration, should be included in the output.
+  using ShouldFormatEntityCallback =
```

Comment:
Elsewhere I think we do `Fn` rather than `Callback` (`ValidateFn`, `BuildDiagnosticFn`, `AppendFn`, `InitializeTargetsFn`, `ContextFnT`)

```suggestion
  using ShouldFormatEntityFn =
```